misc/zpipe_append.c

```diff
@@ -210,7 +213,7 @@ static int inf(FILE *source, FILE *dest, int window_bits, int _flush,
 		return Z_ERRNO;
 
 	out = malloc(CHUNK_o);
-	if (in == NULL) {
+	if (out == NULL) {
```

Comment:
This is just the same as above. "in" already had its shot and is being verified again. Also I included free(in) since it should be allocated at this point.